model-optimizer/mo/middle/passes/fusing/fuse_linear_ops.py

```diff
@@ -84,6 +91,36 @@ def _fuse_mul(graph: Graph, node: Node, fuse_nodes: list, backward: bool = True)
         ch_dim = weights_port.data.get_attr('output_channel_dim' if backward else 'input_channel_dim')
         shape = np.array([weights_port.data.get_shape()[ch_dim]])
 
+        group = fuse_node.soft_get('group', 1)
+        if group != 1 and shape[0] % group != 0:
+            shape *= group
+            # reshape weights [I,O,H,W] -> [I/G,G*O,H,W]
```

Comment:
I expect that `I` is evenly divided by `G`.

@GlebKazantaev , could you comment?